<?xml version="1.0" encoding="UTF-8"?>
<testsuite name=": Regression Test on Pipeline"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="16.445">
  <properties />
            <testcase name="verifyTheDealPageIsDisplayed" time="16.445" classname="com.pack.common.tests.TheDealPageTest">
                <failure type="java.lang.AssertionError"
                  message="The Deal Page title does not match expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: The Deal Page title does not match expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertTrue(Assert.java:42)
  com.pack.common.tests.TheDealPageTest.verifyTheDealPageIsDisplayed(TheDealPageTest.java:26)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
